<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomePageDialog</class>
 <widget class="QDialog" name="HomePageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trout Heaven</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: #13475c;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>65</y>
     <width>296</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	background-color: rgb(9, 43, 56);
	border-radius: 20px
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #dbdbdbe5;
    word-spacing: 10px;
    letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>Trout Heaven</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="statsButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 4px solid #092b38;
    background-color: #13475c;
    color: #dbdbdbe5;
    font-size: 15px;
    outline: none;
    letter-spacing: 1.3px;
    border-radius: 10px;
}

QPushButton:hover {
	border: 5px solid #092b38;
}</string>
   </property>
   <property name="text">
    <string>Statistiche</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="historyTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>301</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPlainTextEdit {
	border-radius: 15px;
	background-color: #13475c;
	border: 6px solid #092b38;
	padding-top: 10px;
	padding-left: 10px;
	color: #dbdbdbe5;
	font-size: 15px;
}</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 4px solid rgb(0, 85, 0);
	background-color: rgb(0, 118, 0);
	color: rgb(220, 220, 220);
    font-size: 15px;
    outline: none;
    letter-spacing: 1.3px;
    border-radius: 10px;
}

QPushButton:hover {
 	border: 5px solid rgb(0, 85, 0);
}</string>
   </property>
   <property name="text">
    <string>Pesca</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 4px solid rgb(127, 0, 0);
	background-color: rgb(184, 0, 0);
    color: #dbdbdbe5;
    font-size: 15px;
    outline: none;
    letter-spacing: 1.3px;
    border-radius: 10px;
}

QPushButton:hover {
	border: 5px solid rgb(127, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scoreBoardButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 4px solid #092b38;
    background-color: #13475c;
    color: #dbdbdbe5;
    font-size: 15px;
    outline: none;
    letter-spacing: 1.3px;
    border-radius: 10px;
}

QPushButton:hover {
	border: 5px solid #092b38;
}</string>
   </property>
   <property name="text">
    <string>Classifica</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusColor">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>25</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	
	background-color: rgb(186, 0, 0);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="myNickLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>450</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #dbdbdbe5;
    word-spacing: 10px;
    letter-spacing: 1px;
	padding-left: 15px;
}</string>
   </property>
   <property name="text">
    <string>You: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="storeBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>340</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 4px solid #092b38;
    background-color: #13475c;
    color: #dbdbdbe5;
    font-size: 15px;
    outline: none;
    letter-spacing: 1.3px;
    border-radius: 10px;
}

QPushButton:hover {
	border: 5px solid #092b38;
}</string>
   </property>
   <property name="text">
    <string>Negozio</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
